### PR TITLE
Package architecture

### PR DESCRIPTION
**Pull request by weerd**
Progress on project architecture and setup, including applying DS/Neue to example page.

@DoSomething/front-end

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,11 +1,50 @@
 {
   "name": "file-api-fallback",
   "version": "0.0.1",
+  "license": "MIT",
+  "author": "Diego Lorenzo",
```
Comment (DFurnes) [inline package.json]: Add your email! :mailbox_with_mail:

File: src/fileApiFallback.js [hunk 2]
```diff
@@ -8,6 +8,6 @@
 
 var FileApiFallback = function (input, container, config) {
 
-
+  
 
 };
```
Comment (DFurnes) [inline src/fileApiFallback.js]: You'll need to export this. So either:
```js
module.exports = FileApiFallback
```

or ES6 syntax:

```js
export default FileApiFallback;
```

Comment (weerd) [inline src/fileApiFallback.js]: Si! :dancer:

**Comment by DFurnes** [comment]
Looks fab! :star2:

**Pull request merged**